### ISSUE TITLE
Can not fetch remote image with https

### ISSUE THREAD

**Issue by molfar** [refile/refile#557]
It seems that refile can not fetch remote image via https protocol. No errors are shown. Tested with image:  https://semena.in.ua/storage/group/e/8/b/5/e8b52ebab5a22eabf60ec2ee61d178a7/original.jpg

**Comment by sobrinho** [comment]
By using this URL on RestClient, I got this:

```
(main) normal> response = RestClient::Request.new(method: :get, url: 'https://semena.in.ua/storage/group/e/8/b/5/e8b52ebab5a22eabf60ec2ee61d178a7/original.jpg', raw_response: true).execute
RestClient::SSLCertificateNotVerified: SSL_connect returned=1 errno=0 state=error: certificate verify failed:
```

Apparently, the certificate is not valid to RestClient (which is used by Refile).

You can force these errors to raise by using `raise_errors: true`, i.e.:

```ruby
class Slide < ApplicationRecord
  attachment :image, raise_errors: true
end
```

I'm not sure how to fix this on RestClient or Down (#554).

You probably need to report this on RestClient and/or Down or try upgrading your Ruby version.

**Comment by sobrinho** [comment]
Apparently, it is a certificate issue itself: https://www.ssllabs.com/ssltest/analyze.html?d=semena.in.ua&hideResults=on

This link will provide more information about how to fix.

If you are not in control of this domain, you would need to skip SSL validation (not recommended unless you know what this means) or to use a proxy.
